src/olympia/devhub/templates/devhub/new-landing/components/navigation.html

```diff
@@ -1,25 +1,27 @@
-<nav class="DevHub-Navigation {% if user_authenticated %}DevHub-Navigation--signedin{% endif %}">
-  <ul>
-    <li>
-      <a href="#" class="Logo {% if user_authenticated %}Logo--secondary{% endif %}">
-        <span>{{ _('Add-on Developer Hub') }}</span>
-      </a>
-    </li>
+<nav class="DevHub-Navigation content {% if user_authenticated %}scheme-light{% else %}scheme-medium{% endif %}">
```

Comment:
True, I was playing around with that and instead using more macros in the less but thanks to less not merging selectors this generates utterly stupid css in some cases. I'd love to just make it `logged-in` and inherit the style in the less via a macro or something.